conformance-suites/1.0.3/conformance/extensions/webgl-compressed-texture-size-limit.html

```diff
@@ -217,11 +218,25 @@
       var dataSize = test.func(t.maxSize + test.sizeStep, t.maxSize + test.sizeStep);
       // this check assumes that each element is 1 byte
       if (dataSize > sharedArrayBuffer.byteLength) {
-        testPassed("Unable to test texture larger than maximum size due to ArrayBuffer size limitations -- this is legal");
+        if (t.target == gl.TEXTURE_CUBE_MAP) {
+          testPassed("Unable to test texture larger than maximum size due to ArrayBuffer size limitations -- this is legal");
+        } else {
```

Comment:
You are right that this technically increases coverage, but the intent has been to maintain the existing coverage as closely as possible. Since max size square textures can't be tested as much after this change, it's been replaced with textures that are large only in one dimension.

If people would rather just remove coverage from the older conformance tests, I'd be okay with that, but if this isn't a big issue to anyone, I could merge the existing version. Any more comments on this?